addon/index.js

```diff
@@ -163,6 +166,18 @@ Object.defineProperty(DS, 'normalizeModelName', {
   value: normalizeModelName
 });
 
-Ember.lookup.DS = DS;
+Object.defineProperty(global, 'DS', {
+  configurable: true,
+  get() {
+    deprecate(
+      'Using the global version of DS is deprecated. Please either import ' +
+        'the specific modules needed or `import DS from \'ember-data\';`.',
```

Comment:
Tiny nitpick: do we want to advertise the `import DS from 'ember.data'`? I think when a pre-addonified-version of ember-data is used, with the shims all the necessary imports should be available via modules, so there is no need for importing `DS` namespace?